packages/overlays/test-suites/OverlayMixin.suite.js

```diff
@@ -19,10 +27,21 @@ function getGlobalOverlayNodes() {
 }
 
 /**
- * @param {{tagString:string, tag: object, suffix?:string}} config
+ * @param {{tagString:string, tag: object, suffix?:string}} [customConfig]
  */
-export function runOverlayMixinSuite({ tagString, tag, suffix = '' }) {
-  describe(`OverlayMixin${suffix}`, () => {
+export function runOverlayMixinSuite(customConfig) {
+  const cfg = {
+    tagString: null,
+    suffix: '',
+    ...customConfig,
+  };
+
+  describe(`OverlayMixin${cfg.suffix}`, () => {
+    class TestOverlay extends OverlayMixin(LitElement) {}
+
+    cfg.tagString = cfg.tagString ? cfg.tagString : defineCE(TestOverlay);
+    const tag = unsafeStatic(cfg.tagString);
```

Comment:
In other words: everywhere we supply both tagString and tag: https://github.com/ing-bank/lion/search?q=runOverlayMixinSuite

So I don't really get how adding `TestOverlay` would make a difference...?

Reply:
The tag somehow doesn't get applied and it shows `lit$316162849$` instead of `test-0`.

All other test-suites have backup class inside the test suite, so this way its also fully aligned.
I will remove the class creation in `./test/OverlayMixin.test.js`, since that is now double.

The slots are also in the test itself, so not needed in the render.

Reply:
Ok, so the difference was explained by incompatible versions of unsafeStatic/staticHtml (from @open-wc and from lit)